packages/checkout-extension/src/handler/ReloadCheckout.ts

```diff
@@ -0,0 +1,13 @@
+import { ExtensionCommandType } from '@bigcommerce/checkout-sdk';
+
+import { CommandHandler } from './CommandHandler';
+
+export const reloadCheckout: CommandHandler = ({ checkoutService, extension }) => {
```

Comment:
Perhaps we could simplify this by doing the following?

```ts
export function createReloadCheckoutHandler(props: CommandHandlerProps): CommandHandler {
    return (command: ReloadCheckoutCommand) => {
        // Reload checkout
    };
}

createReloadCheckoutHandler.commandType = ExtensionCommandType.ReloadCheckout;
```

And then we could do similar to what you've done in `ExtensionService` where we loop through each exported handler and register them:

```ts
Object.values(commandHandlerFactories).forEach((createHandler) => {
    this.handlers[extension.id].push(this.checkoutService.handleExtensionCommand(
        extension.id,
        createHandler.commandType,
        createHandler(props),
    ));
});
```

That way we don't have to call `handleExtensionCommand` in each of the function?